samples/led-more-blinking-lights/led-more-blinking-lights.csproj

```diff
@@ -7,11 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease*" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="..\..\src\devices\Mcp3008\Mcp3008.csproj" />
+    <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease.19078.2" />
+    <PackageReference Include="Iot.Device.Bindings" Version="0.1.0-prerelease.19078.2" />
```

Comment:
Is it necessary / best practice to reference both packages? I would think that I.D.B would be enough and would ensure that the S.D.G version matched.

Reply:
yup, you are right, I'll remove the S.D.G one